man/systemd.exec.xml

```diff
@@ -1631,7 +1631,12 @@ StateDirectory=aaa/bbb ccc</programlisting>
         permit this. Nest <varname>ReadWritePaths=</varname> inside of <varname>ReadOnlyPaths=</varname> in
         order to provide writable subdirectories within read-only directories. Use
         <varname>ReadWritePaths=</varname> in order to allow-list specific paths for write access if
-        <varname>ProtectSystem=strict</varname> is used.</para>
+        <varname>ProtectSystem=strict</varname> is used. Note that <varname>ReadWritePaths=</varname> cannot
+        be used to gain write access to a file system whose superblock is mounted read-only. On Linux, for
```

Comment:
Hmm, is the superblocked *mounted*? I think it'd be more consistent to say "whose superblock is read-only".

Reply:
yeah, it kinda is. By the linux definition at least. I mean, a superblock original is just the first bunch of sectors on disk. But on Linux "superblock" is the data structure it keeps in memory for any file system that is currently mounted. The mount points are then references to that file system. And it's that in-memory object that has the read-only flag set, not the thing on disk. Hence I used the word "mounted", because it is the in-memory thing, not the on-disk thing, i.e. a mount *has* taken place here before we can talk about the thing.

Open to suggestions to say this better?